src/components/quiz/questions.ts

```diff
@@ -0,0 +1,91 @@
+export interface Question {
+  id: number;
+  Question: string;
+  Answers: string[];
+  Answer: number;
+  details: string;
+  url: string;
+}
+
+export const questions: Question[] = [
+  {
+    "id": 1,
+    "Question": "How many moons does Mars have?",
+    "Answers": ["none", "1", "2"],
+    "Answer": 2,
+    "details": "Mars has 2 moons. They are named Phobos, meaning fear and Deimos, which is greek for panik.",
+    "url": "https://svs.gsfc.nasa.gov/vis/a010000/a011300/a011326/marsmoons_1024x576.jpg"
```

Comment:
Prefer no quote marks for object keys:
```suggestion
    questionText: "How many moons does Mars have?",
    answersText: ["none", "1", "2"],
    correctAnswerIndex: 2,
    answerDetailsText: "Mars has 2 moons. They are named Phobos, meaning fear and Deimos, which is greek for panik.",
    questionImageUrl: "https://svs.gsfc.nasa.gov/vis/a010000/a011300/a011326/marsmoons_1024x576.jpg"
```

There's something that may help you rename these! In VSCode you can press "Ctrl + H" to open the "find and replace" tool. You can type `"Question"` in the top and `questionText` in the bottom, and it will replace all occurrences in the file!